std/testing/bench.ts

```diff
@@ -30,6 +30,30 @@ export interface BenchmarkDefinition {
 export interface BenchmarkRunOptions {
   only?: RegExp;
   skip?: RegExp;
+  silent?: boolean;
+}
+
+export interface BenchmarkResult {
+  name: string;
+  totalMs: number;
+  runsCount?: number;
+  runsAvgMs?: number;
+  runsMs?: number[];
```

Comment:
It's not obvious what are those fields, then could use a JS doc

Reply:
@bartlomieju I intended to do the docs after the API was agreed upon.
If not with this: https://github.com/denoland/deno/issues/5841#issuecomment-635855964, I plan to open another PR to do the DOCS